aten/src/ATen/core/function_schema_inl.h

```diff
@@ -84,6 +84,29 @@ inline bool Argument::isBackwardCompatibleWith(
     return true;
 }
 
+// for now, it is same as backward compatible check.
+inline bool Argument::isForwardCompatibleWith(
+    const Argument& old,
+    std::ostream* why_not) const {
+  const Argument* lhs = this;
+  const Argument* rhs = &old;
+  if (!(lhs->name() == rhs->name() && lhs->N() == rhs->N() &&
+        lhs->alias_info() == rhs->alias_info())) {
+    return false;
+  }
+  if (lhs->kwarg_only() && !rhs->kwarg_only()) {
+    return false;
+  }
+  if (!rhs->type()->isSubtypeOfExt(lhs->type(), why_not)) {
```

Comment:
Should this be reversed?

IIUC, `rhs` stands for older version of schema, `lhs` stands for newer version of schema. When `lhs > rhs`, ie newer version of schema allows a type that is a super type of older version of schema, we would have forward compatibility problem. Let me know what you think.

Reply:
Yeah you are right. It should be reversed. Will update